platforms/hyperledger-indy/configuration/samples/network-indyv3.yaml

```diff
@@ -11,80 +15,117 @@ network:
     retry_count: 20                 # Retry count for the checks
     external_dns: disabled           # Should be enabled if using external-dns for automatic route configuration
 
+  # Docker registry details where images are stored. This will be used to create k8s secrets
+  # Please ensure all required images are built and stored in this registry.
+  # Do not check-in docker_password.
   docker:
     url: "docker_url"
     username: "docker_username"
     password: "docker_password"
 
   name: baf
```

Comment:
what is name used for?

Reply:
It's used as the Indy network name (has impact e.g. on paths where the Indy nodes look for crypto files on their local filesystem), @tomas-zigardi please specify this here